### DISCUSSION TITLE
Where to report bug common in all apps?

### DISCUSSION THREAD

**Discussion by simpleguy260** [FossifyOrg/General-Discussion#532] [Q&A]
I have an issue which is common in all apps.
It is when using auto theming based on system theme. It only changes theme when I closes and reopen apps. It creates a couple of issues.
Like, I last used phone/message/contact/clock/calander app in morning (light theme) and gets a call/sms/added contact/alarm/event notification in evening (dark theme). Opening app from notification opens app in light theme. Only way to fix is to close app and open from drawer. 

So where to report it? In all apps? It is due to UI toolkit? It can be reported in fossify-commons (but creating issue there is not allowed)?

**Comment by Aga-C** [comment]
For issues affecting multiple apps or the entire project, check out [general issues](https://github.com/FossifyOrg/General-Discussion/issues).
